### ISSUE TITLE
Thank you for your excellent work. Do you have a pytorch version of the code?

### ISSUE THREAD

**Issue by TJL2018GH** [SuperBruceJia/EEG-DL#4]

**Comment by SuperBruceJia** [comment]
Oh, PyTorch! Intriguing! Currently, a PyTorch version of this code is unavailable. However, you are encouraged to write and implement a PyTorch-version code of this repo, and I'll share it with the community as soon as u finished it. Thanks in advance and have a nice day.

**Comment by TJL2018GH** [comment]
您是东北电力大学的是吗？那咱还是用中文吧，哈哈哈😂😂。有几个模型我可能会用到pytorch，我试试看。你们开源的这个仓库很牛逼！！

Shuyue Jia <notifications@github.com> 于2020年12月4日周五 上午11:07写道：

**Comment by SuperBruceJia** [comment]
Cool dude. Thanks a lot and have a nice day!

**Comment by TJL2018GH** [comment]
Hello, researcher
Recently I am verifying the GCN model. I ran main_GCN.py, I found the
accuracy of the validation set was only 24.8, and after 100 epochs, the
accuracy remained unchanged. And I found that the accuracy rate in your
paper GCNs-Net is very high. I would like to ask what is the reason for
this?
Shuyue Jia <notifications@github.com> 于2020年12月25日周五 下午4:24写道：
